errors/Cargo.toml

```diff
@@ -12,5 +12,10 @@ bevy = { path = ".." }
 workspace = true
 
 [package.metadata.docs.rs]
-rustdoc-args = ["-Zunstable-options", "--cfg", "docsrs"]
+rustdoc-args = [
+  "-Zunstable-options",
+  "--cfg",
+  "docsrs",
+  "--generate-link-to-definition",
+]
 all-features = true
```

Comment:
(not for this PR) these crates are never published, right? surely we can get rid of this.

Reply:
Yup, that can be removed! It has `publish = false`.

Reply:
I've removed the docs.rs metadata for all crates that have `publish = false` set